frontend/src/components/eMIB/constants.js

```diff
@@ -48,3 +48,21 @@ export const actionShape = PropTypes.shape({
   emailCc: PropTypes.string,
   emailBody: PropTypes.string
 });
+
+/* Override margins for the Instuctions and Background pages and nothing else */
+export const instructionStyles = {
```

Comment:
can we make a separate `styleConstants.js` if we want to keep this in a separate file?

Reply:
Changed in c5dba5a and fb3d8c4